src/constants/editor/quill_bubble_css.ts

```diff
@@ -912,7 +912,7 @@ export const quill_bubble_css = (cdn: boolean) =>
 }
 .ql-bubble .ql-tooltip-editor a:before {
   color: #ccc;
-  content: "\\D7";
+  content: "\D7";
```

Comment:
I remember that two back-slashed were required in order to package not crash. ex. "\\D7" instead of "\D7"

Reply:
Thank you for the review.
I corrected the backslashes in quill_bubble_css.ts and quill_snow_css.ts.